api/lib/index.js

```diff
@@ -34,6 +35,20 @@ const lib = {
 
     return false;
   },
+  isRangeOfNumbers: str => {
+    if (typeof str !== 'string') {
+      return false;
+    }
+
+    // will validate 123 or 123+123... or 123-124
+    const numbersRegEx = /^[<>]?(([0-9]+)([+-](?=[0-9]))?)+$/;
```

Comment:
was thinking about this line, is something like "<1000>500" considered a valid input?